### ISSUE TITLE
Update and integrate system monitor plugin into core

### ISSUE THREAD

**Issue by brentgriffin** [dotCMS/core#14764]
Need to update the [system monitor plugin](https://github.com/dotCMS/plugin-system-monitor) to work with the updated ElasticSearch in 5.0 (CountResponse is no longer a class provided by ES).  This updated code needs to be placed in the core rather than in a plugin.  

For reference, this issue specifies original functionality requested:  https://github.com/dotCMS/core/issues/12804

**Comment by brentgriffin** [comment]
**default config settings in dotmarketing-config.properties:**
SYSTEM_STATUS_API_LOCAL_FS_TIMEOUT=1000
SYSTEM_STATUS_API_CACHE_TIMEOUT=1000
SYSTEM_STATUS_API_ASSET_FS_TIMEOUT=1000
SYSTEM_STATUS_API_INDEX_TIMEOUT=1000
SYSTEM_STATUS_API_DB_TIMEOUT=1000
SYSTEM_STATUS_API_IP_ACL=127.0.0.1/32,0:0:0:0:0:0:0:1/128


**Two Ways it can be called and the related responses:**
```
    GET http://localhost:8080/api/v1/system-status

        200 status code if everything is healthy (no content body)
        403 status code if client IP is not permitted via the SYSTEM_STATUS_API_IP_ACL setting (no content body)
        500 status code not expected but always a possiblity (no content body)
        503 status code if any subsystems affecting the frontend are down (no content body)
        507 status code if everything except working index is healthy - frontend is healthy but backend is not(no content body)
```

```
    GET http://localhost:8080/api/v1/system-status?extended

        200 status code unless access is forbidden or unexpected system error (500) - returns content body like:
        {
            "backendHealthy": true,
            "clusterID": "198c8718-2c6d-411b-94ec-578ad88fcb8c",
            "dotCMSHealthy": true,
            "frontendHealthy": true,
            "serverID": "bf0e90f5-048a-478b-8d2b-d22d4d22815c",
            "subsystems": {
                "assetFSHealthy": true,
                "cacheHealthy": true,
                "dbSelectHealthy": true,
                "indexLiveHealthy": true,
                "indexWorkingHealthy": true,
                "localFSHealthy": true
            }
        }
        403 status code if client IP is not permitted via the SYSTEM_STATUS_API_IP_ACL setting (no content body in this case)
        500 status code not expected but always a possiblity (no content body in this case)
```

If SYSTEM_STATUS_API_IP_ACL is set to be empty, there will be no IP restriction.  (Requests from all IPs will be permitted)

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/14847

**Comment by bryanboza** [comment]
Fixed, tested on release-5.0 and now we have this integrated in master
